libs/dart/vault/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ resolution: workspace
 
 dependencies:
   affinidi_tdk_common: ^1.6.0
+  dio: ^5.8.0+1
```

Comment:
Autogenerated [clients](https://github.com/affinidi/affinidi-tdk/blob/main/packages/dart/vault_data_manager/pubspec.yaml#L20) use version `^5.7.0`. Can we use that same minimum version or do you need the latest one?